x/avs/keeper/avs.go

```diff
@@ -187,3 +188,22 @@ func (k Keeper) GetChainIDByAVSAddr(ctx sdk.Context, avsAddr string) (string, bo
 	}
 	return string(bz), true
 }
+
+// GetAllChainIdInfos returns a slice containing all chain id information.
+func (k *Keeper) GetAllChainIDInfos(ctx sdk.Context) ([]types.ChainIDInfo, error) {
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefixAVSAddressToChainID)
+	iterator := sdk.KVStorePrefixIterator(store, nil)
+	defer iterator.Close()
+
+	ret := make([]types.ChainIDInfo, 0)
+	for ; iterator.Valid(); iterator.Next() {
+		avsAddr := strings.ToLower(common.BytesToAddress(iterator.Key()).Hex())
```

Comment:
It seems like using `ToLower` is unnecessary, because we have converted it to lowercase when set, right?

Reply:
After testing, the address generated by common. Bytes. string() may not necessarily be all lowercase letters